### PR TITLE
Updated docs and scripts for rc release

### PR DESCRIPTION
**Pull request by adityapatwardhan**

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

File: docs/FAQ.md [hunk 1]
```diff
@@ -47,9 +47,9 @@ To use the `Microsoft.PowerShell.SDK` NuGet package, declare `PackageReference`
 
 ```xml
 <ItemGroup>
-  <PackageReference Include="Microsoft.PowerShell.SDK" Version="6.0.0-beta.9" />
-  <PackageReference Include="Microsoft.PowerShell.Commands.Diagnostics" Version="6.0.0-beta.9" />
-  <PackageReference Include="Microsoft.WSMan.Management" Version="6.0.0-beta.9"/>
+  <PackageReference Include="Microsoft.PowerShell.SDK" Version="6.0.0-rc" />
+  <PackageReference Include="Microsoft.PowerShell.Commands.Diagnostics" Version="6.0.0-rc" />
+  <PackageReference Include="Microsoft.WSMan.Management" Version="6.0.0-rc"/>
```
Comment (daxian-dbw) [inline docs/FAQ.md]: Don't need to update example content.

File: docker/InstallTarballPackage.sh [hunk 2]
```diff
@@ -5,7 +5,7 @@ set -e
 
 #
 # Example use:
-#     ./InstallTarballPackage.sh "6.0.0-beta.9" "powershell-6.0.0-beta.9-linux-x64.tar.gz"
+#     ./InstallTarballPackage.sh "6.0.0-rc" "powershell-6.0.0-rc-linux-x64.tar.gz"
```
[the remark this answers is not in the document]
Comment (adityapatwardhan) [reply docker/InstallTarballPackage.sh]: People typically copy-paste this.. I believe we should point to the latest. Your thoughts?

File: docs/FAQ.md (hunk 1 above)
Comment (adityapatwardhan) [reply docs/FAQ.md]: People typically copy-paste this.. I believe we should point to the latest. Your thoughts?

File: docker/InstallTarballPackage.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (daxian-dbw) [reply docker/InstallTarballPackage.sh]: We have many other examples in docs, and we don't update the version numbers in them, so I think we should do the same here.

File: docs/FAQ.md (hunk 1 above)
Comment (daxian-dbw) [reply docs/FAQ.md]: There are many other examples in our docs or scripts, and we don't update the version numbers in them, for example, `6.0.0-alpha.9` in https://github.com/PowerShell/PowerShell/blob/master/docs/installation/windows.md.

**Comment by adityapatwardhan** [comment]
@daxian-dbw Updated with CR comments

**Review by daxian-dbw — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -53,20 +53,20 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
...
-[rl-linux-tar]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-beta.9/powershell-6.0.0-beta.9-linux-x64.tar.gz
+[rl-windows-64]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/PowerShell-6.0.0-rc-win-x64.msi
+[rl-windows-86]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/PowerShell-6.0.0-rc-win-x86.msi
+[rl-ubuntu17]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell_6.0.0-rc-1.ubuntu.17.04_amd64.deb
+[rl-ubuntu16]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell_6.0.0-rc-1.ubuntu.16.04_amd64.deb
+[rl-ubuntu14]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell_6.0.0-rc-1.ubuntu.14.04_amd64.deb
+[rl-debian8]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell_6.0.0-rc-1.debian.8_amd64.deb
+[rl-debian9]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell_6.0.0-rc-1.debian.9_amd64.deb
+[rl-centos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-rc/powershell-6.0.0_rc-1.rhel.7.x86_64.rpm
```
Comment (iSazonov) [inline README.md]: We still haven't rewritten `_rc` to `-rc` 😕

**Pull request merged**